doc/getting_started/using_package_manager.rst

```diff
@@ -1,32 +1,30 @@
-.. _getting_started-using_binary:
+.. _getting_started-using_package_manager:
 
 --------------------------------------------------------------------------------
-Using a binary package
+Using a package manager
 --------------------------------------------------------------------------------
 
-For production purposes, we recommend
-`official binary packages <http://tarantool.org/download.html>`_.
+For production purposes, we recommend our
+`official package manager <http://tarantool.org/download.html>`_.
```

Comment:
Well, what does this sentence mean? I understand it prior to the changes but now it's just a gibberish.

Reply:
I agree. My bad. Fixed in 87ab892